### PR TITLE
Push notification

### PR DESCRIPTION
**Pull request by kranjanford**

### CODE REVIEW

File: Example/openxcframework/NotificationViewController.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    @IBAction func networkDropChange(_ sender: UISwitch) {
+        UserDefaults.standard.set(sender.isOn, forKey:"networkDropChange")
+        if sender.isOn {
+        }else{
+            
+        }
+    }
+
+    /*
+    // MARK: - Navigation
```
Comment (mstocke) [inline Example/openxcframework/NotificationViewController.swift]: This commented section can be removed

File: Example/openxcframework/StatusViewController.swift [hunk 2]
```diff
@@ -76,6 +76,8 @@ class StatusViewController: UIViewController, UITableViewDelegate, UITableViewDa
         vm.setCommandDefaultTarget(self, action: StatusViewController.handle_cmd_response)
         // turn on debug output
         vm.setManagerDebug(true)
+       //NotificationCenter.default.addObserver(self, selector: #selector(powerDrop), name: NSNotification.Name("BLEDisconnect"), object: nil)
```
Comment (mstocke) [inline Example/openxcframework/StatusViewController.swift]: This comment can be removed

File: Example/openxcframework/StatusViewController.swift [hunk 3]
```diff
@@ -76,6 +76,8 @@ class StatusViewController: UIViewController, UITableViewDelegate, UITableViewDa
...
+
     }
     override func viewDidAppear(_ animated: Bool) {
         //  let name = UserDefaults.standard.value(forKey: "networkAdress") as? NSString
```
Comment (mstocke) [inline Example/openxcframework/StatusViewController.swift]: This comment can be removed

File: openxcframework/BluetoothManager.swift [hunk 4]
```diff
@@ -319,14 +317,14 @@ open class BluetoothManager: NSObject,CBCentralManagerDelegate,CBPeripheralDeleg
...
   open func centralManager(_ central: CBCentralManager, didDisconnectPeripheral peripheral: CBPeripheral, error: Error?) {
     vmlog("in centralManager:didDisconnectPeripheral:")
-    //vmlog(error!)
+    // update UI if necessary
     let autoOn = UserDefaults.standard.bool(forKey: "autoConnectOn")
     // just reconnect automatically to the same device for now
     if peripheral == openXCPeripheral && autoOn{
       centralManager.connect(openXCPeripheral, options:nil)
       
       // notify client if the callback is enabled
```
Comment (mstocke) [inline openxcframework/BluetoothManager.swift]: This commented block can be removed

File: Example/openxcframework/NotificationViewController.swift (hunk 1 above)
Comment (kranjanford) [reply Example/openxcframework/NotificationViewController.swift]: We need the MARK:,To Do :,  Things in code these are for a purpose.

File: openxcframework/BluetoothManager.swift (hunk 4 above)
Comment (kranjanford) [reply openxcframework/BluetoothManager.swift]: No we need this commented code as this is open source code .User only can understand the line of code for the comment.

**Pull request merged**